src/System.Net.Security/src/System/Net/Security/SslStreamPal.Windows.cs

```diff
@@ -46,9 +46,18 @@ public static byte[] ConvertAlpnProtocolListToByteArray(List<SslApplicationProto
             return Interop.Sec_Application_Protocols.ToByteArray(protocols);
         }
 
-        public static SecurityStatusPal AcceptSecurityContext(ref SafeFreeCredentials credentialsHandle, ref SafeDeleteContext context, SecurityBuffer[] inputBuffers, SecurityBuffer outputBuffer, SslAuthenticationOptions sslAuthenticationOptions)
+        public static SecurityStatusPal AcceptSecurityContext(ref SafeFreeCredentials credentialsHandle, ref SafeDeleteContext context, ArraySegment<byte> input, ref byte[] outputBuffer, SslAuthenticationOptions sslAuthenticationOptions)
```

Comment:
Why ArraySegment, vs Span or Memory if necessary?

Reply:
Can't be span, as we need to be able to extract the original array/offset/count. It could be memory, but memory here doesn't currently buy us anything beyond what arraysegment does, making it somewhat arbitrary which to pick, and as to use memory to get at the underlying array I'd need to ask it for an arraysegment anyway, I just went with arraysegment.

Reply:
Ok